services/web/server/src/simcore_service_webserver/functions/_controller/_functions_controller_rpc.py

```diff
@@ -0,0 +1,207 @@
+from aiohttp import web
+from models_library.api_schemas_webserver import WEBSERVER_RPC_NAMESPACE
+from models_library.functions import (
+    Function,
+    FunctionID,
+    FunctionIDNotFoundError,
+    FunctionInputs,
+    FunctionInputSchema,
+    FunctionJob,
+    FunctionJobCollection,
+    FunctionJobCollectionIDNotFoundError,
+    FunctionJobID,
+    FunctionJobIDNotFoundError,
+    FunctionOutputSchema,
+    RegisteredFunction,
+    RegisteredFunctionJob,
+    RegisteredFunctionJobCollection,
+    UnsupportedFunctionClassError,
+    UnsupportedFunctionJobClassError,
+)
+from models_library.rest_pagination import PageMetaInfoLimitOffset
+from servicelib.rabbitmq import RPCRouter
+
+from ...rabbitmq import get_rabbitmq_rpc_server
+from .. import _functions_repository, _functions_service
+
+router = RPCRouter()
```

Comment:
Q: why do I see this interface as new? I see that it's the one from the previous PR. (just a bit confused)

Reply:
PC asked me to add a service layer, so the previous file got split in 2.